From c626adbf1e9e70e9cf5cd0e30463c52b94c559a7 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 25 Jul 2018 10:16:16 +0300
Subject: [PATCH 4233/5242] MLK-19041: clk: imx8mq: remove
 IMX8MQ_CLK_AUDIO_AHB_DIV from clks_init_on[]

commit  148dcf50c57c045502aae1aada1f432595c0f9cc from
https://source.codeaurora.org/external/imx/linux-imx.git

No need to enable IMX8MQ_CLK_AUDIO_AHB_DIV during imx8mq clock
driver init, so remove it from the clks_init_on[].

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit b13900ec3296f579f54581483858cc053d2bbff3)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8mq.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/imx/clk-imx8mq.c b/drivers/clk/imx/clk-imx8mq.c
index fd133f6..80adc75 100644
--- a/drivers/clk/imx/clk-imx8mq.c
+++ b/drivers/clk/imx/clk-imx8mq.c
@@ -282,7 +282,7 @@
 	IMX8MQ_CLK_NOC_CG, IMX8MQ_CLK_NOC_APB_CG,
 	IMX8MQ_CLK_USB_BUS_CG, IMX8MQ_CLK_NAND_USDHC_BUS_CG,
 	IMX8MQ_CLK_MAIN_AXI_CG, IMX8MQ_CLK_A53_CG,
-	IMX8MQ_CLK_AUDIO_AHB_DIV, IMX8MQ_CLK_TMU_ROOT,
+	IMX8MQ_CLK_TMU_ROOT,
 	IMX8MQ_CLK_DRAM_APB_DIV,
 };
 
-- 
1.7.9.5

